addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1577,6 +1580,26 @@ MockServer.include({
             "active": partner.active,
         };
     },
+    /**
+     * Simulates `/mail/get_suggested_recipients` route.
+     *
+     * @private
+     * @returns {Object} args
+     * @returns {Array}
+     */
+    _mockGetSuggestedRecipient(args) {
+        let partners = this._getRecords(
+            'res.partner',
+            [['id', 'in', args.res_ids]],
+        );
```

Comment:
I know this is a simplification but that looks very confusing. What do you attempt to do here?

Method should at least check for the target `model` & `res_id` and look into a specific field of it. 
Basically mock `_message_get_suggested_recipients` with a default behavior (like python checking for the presence of `user_id` in the target model), and override it if necessary/or make a test that fit the case.